.github/workflows/release-post-merge.yml

```diff
@@ -84,7 +88,7 @@ jobs:
               repo="mongodb/mongodb-atlas-kubernetes-operator-prerelease"
               RELEASE_HELM=false
               CERTIFY=false
-              RELEASE_TO_GITHUB=true
+              RELEASE_TO_GITHUB=false
```

Comment:
Will this become a manual step?

Reply:
Not for the releases, this is the pre-release behaviour. We do not want pre-releases to reach the GitHub releases section on this repo.